app/services/manifest_builder.rb

```diff
@@ -773,6 +774,18 @@ def base_url(file_set)
         nil
       end
     end
+
+    private
+
+      def pyramidal_url(file_set)
```

Comment:
Does this provide the pyramidal_url for objects stored in s3, also? Even though it's the `CantaloupeHelper`?